src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    String BOOKS_TABLE = "books";
```

Comment:
```suggestion
```

Reply:
why? It's comfortable and easy to use one variable in all methods!

Reply:
I've changed it to: private static final String BOOKS_TABLE = "books";